### DISCUSSION TITLE
I haven't get student developer pack benefits

### DISCUSSION THREAD

**Discussion by FenLo** [community/community#131796] [GitHub Education]
### Hi everyone,

I haven't get student developer pack benefits

**Comment by naveenalla3000** [comment]
Go to [GitHub Education](https://github.com/education). Join the program and select your educational institution (make sure your GitHub account is linked with your institute email; if not, add it). Submit any required proof that you belong to the educational institution.
please make correct if it's helpful.

**Reply by FenLo** [reply]
I already accomplish these steps 3 days ago, but my benefits still not accessible

**Reply by naveenalla3000** [reply]
after completing the process it show the status in progress or rejected.
if the status is progress wait for 2 more days to get access.
else re-apply new one.

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
